product_docs/docs/biganimal/release/pricing_and_billing/index.mdx

```diff
@@ -22,8 +22,16 @@ Extreme high availability powered by EDB Postgres Distributed is now available i
 
 ## Cloud infrastructure costs
 
+### BigAnimal's cloud account
+
 If you're connecting directly to BigAnimal through BigAnimal's cloud account, EDB bills you for cloud infrastructure, such as compute, storage, data transfer, monitoring, and logging. You can view monthly estimates for BigAnimal's cloud infrastructure costs while you're creating a cluster. 
 
+A BigAnimal Infrastructure Unit (BIU) is a unit of measure for cloud infrastructure resources, such as compute, storage, and data transfer. The nominal value of one BIU is $1.00. For example, an AWS compute instance priced at $0.192/instance-hour is multiplied by 1 BIU, so that the equivalent BIU cost is $0.192/BIU-hour.
+
+Within the BigAnimal portal, you can see a summary of the estimated charges for the database usage and BIUs.
+
+### Microsoft Azure or AWS account
```

Comment:
I think we should stick with the "using your own cloud account" terminology we have used elsewhere so we don't need to update it when we add a new CSP like GCP.